public/locales/cs/common.json

```diff
@@ -200,5 +200,27 @@
     "android_soon": "Android verze už brzy!",
     "play": "přehrát",
     "new_game": "Nová hra"
+  },
+  "christmas_banner": {
+    "cs": {
+      "christmas_dictionary_title": "Vánoční slovníček",
+      "christmas_dictionary_description": "Vánoce, Vánoce přicházejí a s nimi i náš slovníček s vánoční tematikou.",
+      "christmas_dictionary_link": "/dictionary#vanoce",
+      "witer_dictionary_title": "Zimní slovníček",
+      "winter_dictionary_description": "Bude zima, bude mráz. Slovníček, který se ti bude v zimě hodit.",
+      "witer_dictionary_link": "/kids#kapr",
```

Comment:
This has a typo `witer` vs `winter` and the value seems switched with `christmas_images_link` :)

Reply:
opraveno